fast64_internal/sm64/sm64_geolayout_bone.py

```diff
@@ -72,53 +74,53 @@ def drawGeoInfo(panel, bone):
 	prop_split(col, bone, 'geo_cmd', 'Geolayout Command')
 
 	if bone.geo_cmd in ['TranslateRotate', 'Translate', 'Rotate', 
-		'Billboard', 'DisplayList', 'Scale', 'DisplayListWithOffset']:
+		'Billboard', 'DisplayList', 'Scale', 'DisplayListWithOffset', 'CustomAnimated']:
 		drawLayerWarningBox(col, bone, "draw_layer")
 
 	if bone.geo_cmd == 'Scale':
 		prop_split(col, bone, 'geo_scale', 'Scale')
 	
-	if bone.geo_cmd == 'HeldObject':
+	elif bone.geo_cmd == 'HeldObject':
 		prop_split(col, bone, 'geo_func', 'Function')
 	
-	if bone.geo_cmd == 'Switch':
+	elif bone.geo_cmd == 'Switch':
 		prop_split(col, bone, 'geo_func', 'Function')
 		prop_split(col, bone, 'func_param', 'Parameter')
 		col.label(text = 'Switch Option 0 is always this bone\'s children.')
 		col.operator(AddSwitchOption.bl_idname).option = len(bone.switch_options)
 		for i in range(len(bone.switch_options)):
 			drawSwitchOptionProperty(col, bone.switch_options[i], i)
 
-	if bone.geo_cmd == 'Function':
+	elif bone.geo_cmd == 'Function':
 		prop_split(col, bone, 'geo_func', 'Function')
 		prop_split(col, bone, 'func_param', 'Parameter')
 		infoBox2 = col.box()
 		infoBox2.label(text = 'This affects the next sibling bone in ' +\
 			'alphabetical order.')
 	
-	'''
-	if bone.geo_cmd == 'Function' or \
-		bone.geo_cmd == 'Switch' or \
-		bone.geo_cmd == 'HeldObject':
-		infoBox = col.box()
-		infoBox.label(text = 'For binary, this is a memory address.')
-		infoBox.label(text = 'For C, this is a function name.')
-	'''
-	
-	if bone.geo_cmd == 'TranslateRotate':
+	elif bone.geo_cmd == 'TranslateRotate':
 		prop_split(col, bone, 'field_layout', 'Field Layout')
 
-	if bone.geo_cmd == 'Shadow':
+	elif bone.geo_cmd == 'Shadow':
 		prop_split(col, bone, 'shadow_type', 'Type')
 		prop_split(col, bone, 'shadow_solidity', 'Alpha')
 		prop_split(col, bone, 'shadow_scale', 'Scale')
 	
-	if bone.geo_cmd == 'StartRenderArea':
+	elif bone.geo_cmd == 'StartRenderArea':
 		infoBoxRenderArea = col.box()
 		infoBoxRenderArea.label(text = "WARNING: This command is deprecated for bones.")
 		infoBoxRenderArea.label(text = 'See the object properties window for the armature instead.')
 		prop_split(col, bone, 'culling_radius', 'Culling Radius')
 	
+	elif bone.geo_cmd in {'CustomAnimated', 'CustomNonAnimated'}:
+		prop_split(col, bone.fast64.sm64, 'custom_geo_cmd_macro', 'Geo Command Macro')
+		if bone.geo_cmd == 'CustomNonAnimated':
+			prop_split(col, bone.fast64.sm64, 'custom_geo_cmd_args', 'Geo Command Args')
+		else: # It's animated
+			infobox = col.box()
+			infobox.label(text = "Command's args will be filled with layer, translate, and rotate", icon = "ERROR")
```

Comment:
```suggestion
			infobox.label(text = "Command's args will be filled with layer, translate, and rotate", icon = "INFO")
```
(check out this included-with-blender addon if you never have https://docs.blender.org/manual/en/latest/addons/development/icon_viewer.html )

Reply:
Ohhhh yeah thats a better icon in this scenario, and yeah i heard about this addon but i need to actually use it!

pr updated